app/cypress/integration/community.spec.js

```diff
@@ -140,3 +151,52 @@ describe("Community page (small screen)", () => {
     cy.findByTestId("hmw-community-map").should("not.be.visible");
   });
 });
+
+describe("Community page Show Additional Text", () => {
+  beforeEach(() => {
+    cy.visit("/community/San%20Antonio,%20TX/overview");
+  });
+
+  it("Clicking Show Text switch toggles intro text", () => {
+    cy.findByText("Your Waters: What We Know");
+    cy.findByLabelText("Show Text").click({ force: true });
+    cy.findByText("Your Waters: What We Know").should("not.exist");
+  });
+
+  it("Clicking Show More/Show Less toggles more or less text", () => {
+    const less = "Show less",
+      more = "Show more";
+    //findAllByText is needed due to multiple show more on DOM, but only 1 visbile.
+    cy.findByText(less).should("not.exist");
+    cy.findAllByText(more).filter(":visible").click();
+
+    cy.findByText(less).should("exist");
+    cy.findAllByText(more).filter(":visible").should("not.exist");
+
+    cy.findByText(less).click();
+    cy.findByText(less).should("not.exist");
```

Comment:
Same comment as above.

Reply:
Line 177 needed to not use an alias or an error would be thrown. That line was not changed.